tests/rptest/tests/read_replica_e2e_test.py

```diff
@@ -143,4 +180,34 @@ def test_simple_end_to_end(self, partition_count: int,
 
         # Consume from read replica topic and validate
         self.start_consumer()
-        self.run_validation()
+        self.run_validation()  # calls self.consumer.stop()
+        assert self.redpanda
+        self.redpanda.stop()
+
+        # Run consumer again, this time with source cluster stopped.
+        # Now we can test that replicas do not write to s3.
+        self.start_consumer()
```

Comment:
Doesn't this try to consume from the cluster we just stopped as opposed to the read-replica cluster?

Reply:
I added a comment to `start_consumer()` above, because I also though this was a little confusing. This test overrides the function and replaces the consumer with one targeting the read replica cluster.

Reply:
I see now. Thanks!